srml/support/src/metadata.rs

```diff
@@ -51,61 +53,152 @@ macro_rules! __runtime_modules_to_metadata {
 	(
 		$runtime: ident;
 		$( $metadata:expr ),*;
-		$mod:ident::$module:ident,
+		$mod:ident::$module:ident $(with)+ $($kw:ident)*,
```

Comment:
Is that intentionally that you require `with` at least once, but `kw` can be 0?

Reply:
yes (I send with without content in some cases)